examples/demo_workflow/app.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from time import sleep
+from dapr.ext.workflow.workflow_runtime import WorkflowRuntime
+from dapr.ext.workflow.dapr_workflow_client import DaprWorkflowClient
+from dapr.ext.workflow.dapr_workflow_context import DaprWorkflowContext
+from dapr.ext.workflow.workflow_activity_context import WorkflowActivityContext
```

Comment:
Do we really need so many modules or could we reduce it down to just one or two? Note that we don't need to organize files the same way we do for .NET. Ideally we could just do something like:

```python
from dapr.ext.workflow import DaprWorkflowClient, DaprWorkflowContext, WorkflowActivityContext, WorkflowRuntime
```
...or...
```python
import dapr.ext.workflow as wf
```
And just refer to everything from the `wf.` prefix, like `wf.DaprWorkflowContext`, etc.

Reply:
We could import those into dapr.ext.workflow's __init__.py exactly as the code about so they could be imported as you mentioned. Agree it would make the existing and future code simpler.

Reply:
Done. Please check now.

Reply:
In recent validate examples, somehow import in app.py is failing.
Everything seems working on my local though.
https://github.com/dapr/python-sdk/actions/runs/5020485194/jobs/9001991467#step:17:279

trying to understand that what can be the issue, is somehow dapr-ext-workflow build used in validate examples, not consisting of my __init__.py changes
@berndverst Any suggestions here?